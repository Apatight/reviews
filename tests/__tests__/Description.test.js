import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import ReactDOM from 'react-dom';
import Description from '../../client/components/Description.jsx'

Enzyme.configure({ adapter: new Adapter() });


test('render a label', () => {
    const wrapper = shallow(
        <Description/>
    );
    wrapper.instance().props
    expect(wrapper).toMatchSnapshot();
});
