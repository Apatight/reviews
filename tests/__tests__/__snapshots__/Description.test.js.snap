// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a label 1`] = `
<div
  className="reviews-container"
>
  <div
    className="description-title"
  />
  <div
    className="description-type"
  >
     
     Restaurant
  </div>
  <div
    className="description-details-container"
  >
    <div
      className="description-details"
    />
    <div
      className="desription-details-seperator"
    >
      ·
    </div>
    <div
      className="description-details"
    >
      $
    </div>
  </div>
  <div
    className="logo-container"
  >
    <img
      className="apateez-logo"
      src="https://s3-us-west-1.amazonaws.com/apateezassets/apateez-logo-small-red.jpeg"
    />
    <div
      className="description-divider"
    />
  </div>
  <div
    className="description-apateez-header"
  >
    THE APATEEZ REVIEW
  </div>
  <div
    className="description-ratings-container"
  >
    <div
      className="description-ratings-food"
    >
      4.6
      <div
        className="description-ratings-label"
      >
        FOOD
      </div>
    </div>
    <div
      className="description-ratings-decor"
    >
      4.0
      <div
        className="description-ratings-label"
      >
        DECOR
      </div>
    </div>
    <div
      className="description-ratings-service"
    >
      4.4
      <div
        className="description-ratings-label"
      >
        SERVICE
      </div>
    </div>
  </div>
  <div
    className="description-body"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<b>\\"Inventive”</b> yet <b>“approachable”</b>, this Hayes Valley <b>“destination”</b> by Evan and Sarah Rich <b>“wows”</b> with its <b>“unique”</b>, <b>“delectable”</b> Californian dishes matched by a <b>“killer wine list”</b> and <b>“warm”</b> service in a minimalist (and <b>“loud”</b>) space adorned with salvaged barn wood; it's <b>“hard to get into”</b> (though walk-ins can sit at the bar or the communal table) and the <b>“pricey”</b>, <b>“locally sourced”</b> menu <b>“changes”</b> often, but <b>“if the stars align, you will leave sockless\\"</b>.",
      }
    }
  />
</div>
`;
